RecordCollection/Views/Albums/Show.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@model AlbumViewModel
+
+<h1>Album Details</h1>
+
+<div>
```

Comment:
I would like to split this div into 2 divs, one for the album info through line 17 and one for the related ablums on lines 19-25. I would also like to add some indentation in these divs and the if statements. These are small formatting changes, but I think they will improve the readability of this view.